src/main.py

```diff
@@ -76,13 +78,22 @@
     help="gcp project where firestore db lives",
     **click_kwargs,
 )
+@click.option(
+    "--log-level",
+    envvar="LOG_LEVEL",
+    type=click.Choice(["0", "1", "2", "3"]),
```

Comment:
This numbering scheme was a little confusing to me at first since I hadn't seen it before. I think it's fine now that I see what you did down below (looks useful, and I'm not sure how else I'd separate out debug and debug-imports). For my own knowledge, does this conform to a standard, or implicitly. reference other logging systems?

Reply:
I've seen numeric log levels before, but I wouldn't say it's a "standard" or anything.  Normally this is the sort of thing I'd implement with distinct switches for each, most likely `-q` `-v`, and `-vv`.   But I found wasn't sure how click would handle `-vv` as distinct from `-v`, plus from what I read there's no way to tell click that arguments are mutually exclusive without implementing a custom class for it.

(I'm not a click expert, just spend some time reading docs last night trying to see if I could make it work.)

So to avoid having potentially three different flags, or further complicating the argument parsing with manual logic checking the sanity of the verbosity flags, this seemed like the most direct route.

I definitely don't mind switching it to using discrete flags (and using long options instead of short ones, to keep with the existing pattern) if it feels like better choice to you.

Reply:
There's sort of a way to do this with click. https://click.palletsprojects.com/en/7.x/options/#counting

I think `q` and `v` would still need to be separate options though

Reply:
And the mutual exclusion of them would have to be 'done in post'

Reply:
Ah, neat, good to know!

Since you approved the PR as is, I think I'll go ahead and merge with what we have rather than re-write to use distinct args and add the exclusion logic, if that's ok.   But we know how to do it in the future if we need/want to.  😀

Reply:
Sure!